File: GAVS.AllocationSystem/GAVS.AllocationSystem.Model/CSP/Tables/AUDITEE_ACCEPTANCE.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace GAVS.AllocationSystem.Model.CSP
{
    public class AUDITEE_ACCEPTANCE
    {
        public int ID { get; set; }
        public int FINDING_ID { get; set; }
        public string STATUS { get; set; }
        public string REMARKS { get; set; }
        public DateTime CREATED_DATE { get; set; }
        public string CREATED_BY { get; set; }
        public DateTime UPDATED_DATE { get; set; }
        public string UPDATED_BY { get; set; }
        public bool ISACTIVE { get; set; }
        public string UNIQUE_ID { get; set; }

        public bool ISSUBMITTED { get; set; }
        [NotMapped]
        public int AUDIT_ID { get; set; }


    }
}

File: csp-angular19/src/app/models/auditee-acceptance.ts
/**
 * Auditee Acceptance Model
 * Used for auditor/auditee acceptance or rejection of findings
 */
export class AuditeeAcceptance {
  id: number = 0;
  findinG_ID: number = 0;
  status: string = '';
  remarks: string = '';
  createD_BY: string = '';
  updateD_BY: string = '';
  createD_DATE: Date = new Date();
  updateD_DATE: Date = new Date();
  isactive: boolean = true;
  issubmitted: boolean = false;
  audit_ID: number = 0;
  iS_AUDITOR_ACCEPT: boolean = false;
  disablE_CAPA: boolean = false;
}
